### PR TITLE
[DOCFIX] Doc improvements

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/fs/command/MountCommand.java [hunk 1]
```diff
@@ -124,8 +124,7 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "mount [--readonly] [--shared] [--option <key=val>] <alluxioPath> <ufsURI>\n"
-            + "mount";
+    return "mount [--readonly] [--shared] [--option <key=val>] <alluxioPath> <ufsURI> mount";
```
Comment (calvinjia) [inline shell/src/main/java/alluxio/cli/fs/command/MountCommand.java]: What's the trailing `mount` for?

Comment (aaudiber) [reply shell/src/main/java/alluxio/cli/fs/command/MountCommand.java]: ... fixed

File: docs/en/Running-Hadoop-MapReduce-on-Alluxio.md [hunk 2]
```diff
@@ -54,17 +54,6 @@ installation:
 This will allow your MapReduce jobs to recognize URIs with Alluxio scheme (i.e., `alluxio://`) in
 their input and output files.
 
-Optionally, modify `$HADOOP_CLASSPATH` by changing `hadoop-env.sh` also in `conf` directory to
```
Comment (apc999) [inline docs/en/Running-Hadoop-MapReduce-on-Alluxio.md]: why remove this part?

Comment (aaudiber) [reply docs/en/Running-Hadoop-MapReduce-on-Alluxio.md]: I tried doing this with hadoop 2.7.2 but the classpath change wasn't getting picked up by the the executors. The approaches described in the next section (-libjars and and distributing the client jar) work better.

Comment (apc999) [reply docs/en/Running-Hadoop-MapReduce-on-Alluxio.md]: ack

**Pull request merged**